index.html

```diff
@@ -17,7 +19,10 @@ <h3 class="subtitle">Take a quiz to find your dream job!</h3>
             Do you want to be a programmer? Or maybe you have some experience and want to find a better job? Take a quiz to test your skills and find your dream job! At first, you have to choose your path. Then answer some questions to confirm your knowledge level and see the best job offers for you.
             </p>    
         </div> 
-        <button class="btn">Start</button>
-    </main>
+        <a href="./src/pages/path.html">
+            <button class="btn">Start</button>
+        </a>
```

Comment:
If this is a link you should just use <a> tag and style it. <button> tags are for actions like validating form, showing modal. You don't need to use `onClick` so you should not use <button> tag.
https://medium.com/design-code-repository/a-vs-button-b859547cae4d

Reply:
Done. Placed: 
"<a href="./src/pages/path.html" class="btn">Start</a>" instead the above.